src/Stratis.Bitcoin.IntegrationTests.Common/Extensions/FullNodeTestBuilderExtension.cs

```diff
@@ -58,6 +59,27 @@ public static IFullNodeBuilder MockIBD(this IFullNodeBuilder fullNodeBuilder)
             return fullNodeBuilder;
         }
 
+        public static IFullNodeBuilder UseTestChainedHeaderTree(this IFullNodeBuilder fullNodeBuilder)
```

Comment:
There is an extension called ReplaceService

Reply:
Actually no, we will need to extend ReplaceService to take in a Type as the the replacement.